src/Microsoft.CorrelationVector.UnitTests/CorrelationVectorTests.cs

```diff
@@ -164,7 +167,7 @@ public void SimpleIncrementCorrelationVectorTest()
         [TestMethod]
         public void SimpleExtendCorrelationVectorTest()
         {
-            var correlationVector = new CorrelationVector();
+            CorrelationVector correlationVector = new CorrelationVectorV1();
```

Comment:
let's keep things consistent and use var instead of the type

Reply:
var interprets it as CorrelationVectorV1 which then causes an error on line 175 Cannot implicitly convert type 'Microsoft.CorrelationVector.CorrelationVector' to 'Microsoft.CorrelationVector.CorrelationVectorV1'.